packages/angular/cli/src/utilities/completion.ts

```diff
@@ -88,7 +88,7 @@ Appended \`source <(ng completion script)\` to \`${rcFile}\`. Restart your termi
         ' Angular CLI. For autocompletion to work, the CLI will need to be on your `$PATH`, which' +
         ' is typically done with the `-g` flag in `npm install -g @angular/cli`.' +
         '\n\n' +
-        'For more information, see https://angular.io/cli/completion#global-install',
+        'For more information, see https://angular.dev/cli/completion#global-install',
```

Comment:
This doesn't seem to be pointing to the right section of the doc. (Wrong anchor?)

Reply:
There is no anchor (yet) in CLI docs. I'm planning to make that change on the dev-infra part.